### DISCUSSION TITLE
MRTK3 UX / Data Binding examples missing

### DISCUSSION THREAD

**Discussion by DenisTschitschenkow** [microsoft/MixedRealityToolkit-Unity#11317] [Q&A] [answered]
In the documentation for MRTK3 you see alot of references to Examples scenes for UX and DataBinding. 
In my case those referenced example scenes are nowhere to be found. 
Not in the packages, no links to the "template Project".

One is speaking of UX example scenes like "[CanvasUITearSheet.unity](https://learn.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/mrtk3-uxcomponents/packages/uxcomponents/mixed-reality-design-language#mixed-reality-design-language-in-mrtk3)"

In the Data Binding documentation they speak about

[DataBindingExamples.scene]((https://learn.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/mrtk3-data/packages/data/overview#assetsdatabinding-examplescenesdatabindingexamplesscene))

It would be great for these examples to exist in the Feature Tools Packages or at the very least be linked in the documentation. 

I have tried to get into MRTK3 a few times now and always stop at the lack of shown examples.

Best Regards,
MrMixedReality

**Comment by Zee2** [comment, marked as answer]
Hi @DenisTschitschenkow, the example project is in the `UnityProjects` folder, located here: https://github.com/microsoft/MixedRealityToolkit-Unity/tree/mrtk3/UnityProjects

All of the sample scenes are in the `Assets/Scenes` folder, except for the binding/theming scenes which are in the `Data Binding Example` folder.

Here's the documentation that describes the location of the sample scenes: https://learn.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/mrtk3-overview/scenes

We can add more of these docs to more places, like the README, if that helps.
